app/Controllers/feedController.php

```diff
@@ -480,7 +496,7 @@ public static function actualizeFeed(int $feed_id, string $feed_url, bool $force
 						if (strcasecmp($existingHash, $entry->hash()) !== 0) {
 							//This entry already exists but has been updated
 							//Minz_Log::debug('Entry with GUID `' . $entry->guid() . '` updated in feed ' . $feed->url(false) .
-								//', old hash ' . $existingHash . ', new hash ' . $entry->hash());
```

Comment:
formatting changes only; unfortunately these "escaped" my checks and were committed